component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/duo/DuoAuthenticator.java

```diff
@@ -116,6 +129,28 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
         }
     }
 
+    /**
+     * Check if the tenant domain should be appended or not.
+     *
+     * @param authenticatorProperties the authenticator properties
+     * @return True if the tenant domain should not be appended.
+     */
+    private boolean isDisableTenantDomainToUserName(Map<String, String> authenticatorProperties) {
+
+        return Boolean.parseBoolean(authenticatorProperties.get(DuoAuthenticatorConstants.TENANT_DOMAIN));
+    }
+
+    /**
+     * Check if the user store domain should be appended or not.
+     *
+     * @param authenticatorProperties the authenticator properties
+     * @return True if the user store domain should not be appended.
+     */
+    private boolean isDisableUserStoreDomainToUserName(Map<String, String> authenticatorProperties) {
```

Comment:
Shall we make isDisableTenantDomainToUserName and isDisableUserStoreDomainToUserName method names more readable?

Reply:
What about to change the method name to something as follows:
isDisableTenantDomain**In**UserName
isDisableUserStoreDomain**In**UserName